src/flame/variations/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  linear, sinusoidal, spherical, swirl,
+  randomDisk, gaussian, horseshoe, polar,
+  handkerchief, heart, disc, spiral, hyperbolic,
+  diamond, exVar, julia, bent, fisheye, eyefish,
+  exponential, power, cosine
+} from "./simpleVariations";
+import { grid } from './parametric/grid'
+import { pie } from './parametric/pie'
+import { blob } from './parametric/blob'
+import { pdjVar } from './parametric/pdj'
+import { waves, rings, fan, popcorn } from "./dependentVariations";
+import { Infer } from "typegpu/data";
+import { ParametricVariation } from "./types";
+
+export const transformVariations = {
+  linear,
+  sinusoidal,
+  spherical,
+  swirl,
+  popcorn,
+  pie,
+  randomDisk,
+  gaussian,
+  grid,
+  horseshoe,
+  polar,
+  handkerchief,
+  heart,
+  disc,
+  spiral,
+  hyperbolic,
+  diamond,
+  exVar,
+  julia,
+  bent,
+  waves,
+  fisheye,
+  eyefish,
+  exponential,
+  power,
+  cosine,
+  rings,
+  fan,
+  blob,
+  pdjVar,
+}
+
+export type TransformVariation = keyof typeof transformVariations
+export const variationTypes = Object.keys(transformVariations)
```

Comment:
missing `as TransformVariation[]`

Reply:
actually, this would be even better: ```
const variationTypes = Object.keys(transformVariations)
export function isVariationType(
  maybeType: string,
): maybeType is TransformVariation {
  return variationTypes.includes(maybeType)
}
```

Reply:
Fixed, works now